README.md

```diff
@@ -540,6 +540,19 @@ Any dependency used in any KIE project must fulfill these hard requirements:
     * We don't expect you to check this manually:
     The victims enforcer plugin will automatically fail the build if a known bad dependency is used.
 
+* **The sources are publicly available**
+
+    * We may need to rebuild the dependency from sources ourselves in future. This may be in the rare case when
+      the dependency is no longer maintained, but we need to fix a specific CVE there. The other reason is that
+      productisation needs to be able to easily rebuild the dependency internally.
+
+    * Make sure to link the source repository in the PR which is adding the dependency.
```

Comment:
I'd change this to something like "Make sure the dependency's pom.xml has a source repository link, so a scm tag".
Having it our PR's is useless because productization won't know about and won't go through all our PR's to find it.